lib/bolt/outputter/human.rb

```diff
@@ -665,6 +665,12 @@ def print_plan_result(plan_result)
           print_container_result(value.result)
         when Bolt::ResultSet
           print_result_set(value)
+        when Bolt::Result
+          print_result(value)
+        when Bolt::ApplyResult
+          print_apply_result(value)
+        when Bolt::Error
+          print_error(value)
```

Comment:
`print_error` only prints the message, which means we lose the error location under `details`. I think this info is helpful for users and should be available using human output.

Reply:
Hmm, I think often times that info is already in the message:
```
lucy ~/githubs/bolt (hack/better-output) $ be bolt plan run bolt::error
Starting: plan bolt::error
Finished: plan bolt::error in 0.0 sec
Evaluation Error: Unknown variable: 'target'. (file: /home/lucy/githubs/bolt/plans/error.pp, line: 4, column: 30)
  (file: /home/lucy/githubs/bolt/plans/error.pp, line: 4, column: 30)
```

But can't hurt to have it twice.